mule-apikit-tools/mule-apikit-scaffolder/src/test/java/org/mule/tools/apikit/output/MuleConfigGeneratorTest.java

```diff
@@ -44,27 +44,27 @@
 
 public class MuleConfigGeneratorTest {
```

Comment:
Why all the class is commented? If there is an issue why not use @Ignore at level class??

Reply:
This tests shouldn't be ignored, I miss this, will take a look on this